src/components/Navbar/links.tsx

```diff
@@ -10,6 +10,7 @@ export const navLinks: NavLink[] = [
   { title: "Events", slug: "events" },
   { title: "Visit Us", slug: "visit" },
   { title: "Contact Us", slug: "contact" },
+  { title: "Nixon", slug: "profile", alt: true },
```

Comment:
Can you try to explain what's the purpose of `slug` and `alt`

Reply:
A slug is the unique identifying part of a web address, typically at the end of the URL. The alt attribute specifies an alternate text for an image, if the image cannot be displayed. (setelah gw searching)

Reply:
(not really true for alt) for slug, yes , the alt here itu refers to alternative style since there are two styling for headers

Reply:
oh okkok